app/assets/javascripts/services/deploy_helpers_factory.js

```diff
@@ -0,0 +1,26 @@
+samson.factory('DeployHelper', function($window, $log, Deploys) {
```

Comment:
idk ... do we really need infinite scrolling ... maybe just pagination ... also makes it easier to look at super old stuff and to share urls